src/test-routes/formdata.ts

```diff
@@ -14,4 +15,11 @@ app.category("vanilla", () => {
       },
     };
   });
+  app.post("/formsdataurlencoded/pet/add", "UpdatePetWithForm", (request) => {
+    validateHeader(request.originalRequest, "content-type", "application/x-www-form-urlencoded");
```

Comment:
I would add this to the `request-expectation.ts` so instead you can use those method `request.expect.validateHeader`

for `validateBodyEquals` below you should already be able to.

Reply:
Should all methods from `request-validations` go there (not as part of this or)?

Reply:
It appears that RequestExpectation is just scoped trampolines calling those methods. I'll just add one for the header.